integration_test.go

```diff
@@ -83,7 +82,7 @@ func testCartographerInternalState(t *testing.T, svc slam.Service, dataDir strin
 	test.That(t, err, test.ShouldBeNil)
 }
 
-func integrationtestHelperCartographer(t *testing.T, mode slam.Mode) {
+func integrationtestHelperCartographer(t *testing.T, mode slamTesthelper.Mode) {
```

Comment:
@kkufieta this is using the slam_library remnants from the slam repo's test helper